third_party/blink/renderer/core/farbling/brave_session_cache.cc

```diff
@@ -124,6 +139,49 @@ bool AllowFontFamily(ExecutionContext* context,
   return true;
 }
 
+int MaybeFarbleScreenInteger(ExecutionContext* context,
+                             brave::FarbleKey key,
+                             int spoof_value,
+                             int min_value,
+                             int max_value,
+                             int defaultValue) {
+  if (!brave::BlockScreenFingerprinting(context)) {
+    return defaultValue;
+  }
+  BraveSessionCache& cache = BraveSessionCache::From(*context);
+  return cache.FarbledInteger(key, spoof_value, min_value, max_value);
+}
+
+bool BlockScreenFingerprinting(ExecutionContext* context) {
+// Don't apply this protection on mobile.
+#if BUILDFLAG(IS_ANDROID) || BUILDFLAG(IS_IOS)
```

Comment:
if this doesn't apply to ios and android them why don't we disable kBraveBlockScreenFingerprinting for ios and android? OS guards should generally only be used for os-specific code, not for OS "branching"

Reply:
fixed